chapters/config.Rmd

```diff
@@ -546,6 +549,8 @@ handle a configuration file in INI rather than YAML format.
 1.  Which file format do you find easier to work with?
 2.  What other factors should influence your choice of a configuration file syntax?
 
+Note: we will be referencing the code in YAML format in the rest of this book.
```

Comment:
I don't know what "referencing the code in YAML format" means in this context.

Reply:
This was my attempt at disambiguating required vs optional exercises. Changed this to "the code modified in this exercise is not required for the rest of the book."